LoopKit/Service.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Service.swift
+//  LoopKit
+//
+//  Created by Darin Krauss on 5/17/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+
+public protocol ServiceDelegate: AnyObject {
+
+    /// Informs the delegate that the specified service was updated.
```

Comment:
What aspect of the service updated?  i.e. what would the delegate do in response to this message?

Reply:
Updated protocol documentation with further detail.